test/parallel/test-assert-fail.js

```diff
@@ -67,5 +67,5 @@ common.expectsError(() => {
 // The stackFrameFunction should exclude the foo frame
 assert.throws(
   function foo() { assert.fail('first', 'second', 'message', '!==', foo); },
-  (err) => !/foo/m.test(err.stack)
+  (err) => !/foo(?!\/)/m.test(err.stack)
```

Comment:
Oh sorry, I just saw that you did in fact mention that, my apologies, I will edit it for:

(whitespace)at foo

Thank your for your feedback!